### ISSUE TITLE
Provide page zoom feature

### ISSUE THREAD

**Issue by Jermolene** [Jermolene/TiddlyDesktop#30]
As requested here:

https://groups.google.com/d/topic/tiddlywikidev/F9Ny7Vsahrs/discussion

**Comment by tobibeer** [comment]
There is no more **hosts.html**, where to add the styles these days for https://github.com/Jermolene/TiddlyDesktop/issues/30#issuecomment-67783219?

**Comment by Jermolene** [comment]
See #72 for a possible way to implement this.

**Comment by 38569459** [comment]
Any process about this?

**Comment by enricllagostera** [comment]
Yes, I'd really appreciate this functionality. It helps a lot for using TiddlyDesktop in different contexts / screens without having to manually change CSS styles all the time. I would like to try to implement it but I haven't learned yet how to create plugins yet.

**Comment by mateuszwilczek** [comment]
I have created a tweak that binds keyboard shortcuts `ctrl/cmd` `shift` `+/-/0` to pseudo zoom in/out/reset zoom in TiddlyDesktop. It works by changing the px sizes as defined in vanilla theme tweaks. It has its limitations, so it's not a solution to this issue, but it works well enough for me. See it at https://wilk-tweaks.tiddlyhost.com (or [permalink to demo of this pseudo zoom](https://wilk-tweaks.tiddlyhost.com/#Pseudo%20zoom%20for%20TiddlyDesktop:%5B%5BPseudo%20zoom%20for%20TiddlyDesktop%5D%5D%20About)).
